frontend/src/pages/SubProjects/actions.js

```diff
@@ -185,6 +188,11 @@ export function showProjectAssignees() {
 }
 
 export function assignProject(projectId, projectDisplayName, assigneeId, assigneeDisplayName, showLoading = false) {
+  const isDataValid = validateDataWithSchema({ projectId, projectDisplayName, assigneeId, assigneeDisplayName }, 'assignProject');
```

Comment:
This validation is not necessary. We need to check the payload of the confirmation required action.